src/ocamlorg_frontend/components/footer.eml

```diff
@@ -83,29 +91,41 @@ let render () =
               </li>
 
               <li>
-                <a href="<%s Url.releases %>" class="text-base leading-6 text-lighter">Releases</a>
+                <a href="<%s Url.exercises %>" class="text-base leading-6 text-lighter">Exercises</a>
+              </li>
+
+              <li>
+                <a href="<%s Url.papers %>" class="text-base leading-6 text-lighter">Papers</a>
+              </li>
+
+              <li>
+                <a href="<%s Url.playground %>" class="text-base leading-6 text-lighter">OCaml Playground</a>
               </li>
             </ul>
           </div>
         </div>
-        <div class="md:grid md:grid-cols-2 md:gap-8">
+        <div class="md:grid md:grid-cols-2 md:gap-16">
```

Comment:
```suggestion
        <div class="md:grid md:grid-cols-2 md:gap-8">
```

Why would we change this?

Reply:
because of the gap between the two divs 
i feel it should be increased becaused **community**  changed to Ecosystem & Community
i can make changes to that if it is not okay

Reply:
I think you're right: "Ecosystem & Community" is too long to fit here properly. Let's reduce the caption to "Ecosystem", then it will fit with the old gap.

Reply:
> I think you're right: "Ecosystem & Community" is too long to fit here properly. Let's reduce the caption to "Ecosystem", then it will fit with the old gap.

okay 
changes will made right away